### ISSUE TITLE
508-defect-3 [SCREENREADER]: upon logging in, screen reader announces incorrect information

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#24861]
[508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**VFS Point of Contact:** Angela

## User Story or Problem Statement:
As a screen reader user, I expect spoken alerts to accurately inform me as to what is happening with a site I'm visiting.

## Details
When I log onto my VA dashboard using one of the provided test accounts, JAWS announces that it is loading an application. In fact, it is loading general information which may include a list of in-progress applications.

## Acceptance Criteria
- [ ] JAWS announces something to the effect of "loading your account," or "loading your dashboard."

**Comment by ajakabcin** [comment]
@AngelaFowler82 I'm sharing some follow-up thoughts/questions from @erikphansen on this issue:

1. We think this may be a site-wide issue with the react app. Can you confirm that?
2. When the new spinners come on are they not being read? Because they _should_ be...

**Comment by erikphansen** [comment]
@AngelaFowler82 As AJ said above, I think this might be a site-wide issue.

When you first navigate to My VA, we show a loading spinner with the label "please wait while we load the application for you". This is the loading spinner that is shown while the page is loading the My VA React app. And it is the same loading spinner we show when loading any of the React apps that exist throughout VA.gov.

Once the My VA React app has loaded, that loading spinner is replaced with a new loading spinner with the label "Loading your information".

This is the same experience user's get when visiting their Profile.

- Can you confirm that other React applications like the Profile or Health Care Application do not have this problem?
- Does JAWS only read the first loading spinner? I would think it should read the second "Loading your information" loading spinner when it comes on screen.

**Comment by AngelaFowler82** [comment]
Thank you for following up on this. Upon further testing, I was unable to reproduce the problem. I have updated JAWS since I wrote this ticket, and now neither spinner is read automatically. This seems consistent regardless of the point at which I sign in. Since the problem does not occur with NVDA, I wonder if it resolved itself when I updated JAWS? I would question whether the first spinner is necessary, but as you say that's a sitewide issue.
